File: sdks/node/examples/agent_with_mcp.ts
import { Agent } from '../src/'

const agent = new Agent({
  name: 'A dummy Agent',
  model: 'gpt-4',
  preamble:
    'You are a calculator here to help the user perform arithmetic operations. Use the tools provided to answer the user question.',
  mcpConfigPath: 'servers_config.json',
})
console.log(agent.prompt('Calculate 10 - 3'))

File: sdks/python/examples/agent_with_mcp.py
from alith import Agent

agent = Agent(
    name="Calculator Agent",
    model="gpt-4o-mini",
    preamble="You are a calculator here to help the user perform arithmetic operations. Use the tools provided to answer the user's question.",
    mcp_config_path="servers_config.json",
)
print(agent.prompt("Calculate 10 - 3"))
